src/goto-harness/function_harness_generator_options.h

```diff
@@ -9,13 +9,10 @@ Author: Diffblue Ltd.
 #ifndef CPROVER_GOTO_HARNESS_FUNCTION_HARNESS_GENERATOR_OPTIONS_H
 #define CPROVER_GOTO_HARNESS_FUNCTION_HARNESS_GENERATOR_OPTIONS_H
 
+#include "common_harness_generator_options.h"
+
 #define FUNCTION_HARNESS_GENERATOR_FUNCTION_OPT "function"
-#define FUNCTION_HARNESS_GENERATOR_NONDET_GLOBALS_OPT "nondet-globals"
-#define FUNCTION_HARNESS_GENERATOR_MIN_NULL_TREE_DEPTH_OPT "min-null-tree-depth"
-#define FUNCTION_HARNESS_GENERATOR_MAX_NONDET_TREE_DEPTH_OPT                   \
-  "max-nondet-tree-depth"
-#define FUNCTION_HARNESS_GENERATOR_MIN_ARRAY_SIZE_OPT "min-array-size"
-#define FUNCTION_HARNESS_GENERATOR_MAX_ARRAY_SIZE_OPT "max-array-size"
+#define COMMON_HARNESS_GENERATOR_NONDET_GLOBALS_OPT "nondet-globals"
```

Comment:
It seems surprising that this option isn't in `common_harness_generator_options.h`?!

Reply:
Sorry, my bad. In fact, it's the other way around. This option is `function-harness` specific and will be renamed.